content/en/docs/setup/additional-setup/external-istiod/index.md

```diff
@@ -0,0 +1,309 @@
+
+### API Server Access
+
+The API Server in the remote config cluster must be accessible to the external control plane cluster. Many cloud providers make API Servers publicly accessible via network
+load balancers (NLBs). If the API Server is not directly accessible, you will
+have to modify the installation procedure to enable access. For example, the
+[east-west](https://en.wikipedia.org/wiki/East-west_traffic) gateway used in
+the multi-network and primary-remote configurations could also be used
+to enable access to the API Server.
+
+## Environment Variables
+
+This guide will refer to two clusters named `external_cluster` and `remote_cluster`. The following environment variables will be used throughout to simplify the instructions:
+
+Variable | Description
+-------- | -----------
+`CTX_EXTERNAL_CLUSTER` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the external control plane cluster.
+`CTX_REMOTE_CLUSTER` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the remote config cluster.
+`EXTERNAL_ISTIOD_ADDR` | The external host name for the `remote_cluster` to access the external istiod.
+`SSL_SECRET_NAME` | The secret name used to access the ingress gateway on the external control plane cluster.
+
+For example:
+
+{{< text bash >}}
+$ export CTX_EXTERNAL_CLUSTER=external_cluster
+$ export CTX_REMOTE_CLUSTER=remote_cluster
+$ export REMOTE_ISTIOD_ADDR=myexternal-istiod.cloud.com
+$ export SSL_SECRET_NAME = myexternal-istiod-secret
+{{< /text >}}
+
+## Setup
+
+### Setup the external control plane cluster
+
+Create the Istio configuration for `external_cluster`, using the default profile with the following ports on the ingress gateway to expose the external istiod:
+
+{{< text bash >}}
+$ cat <<EOF > external-cp.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+ namespace: istio-system
+spec:
+ components:
+   ingressGateways:
+     - name: istio-ingressgateway
+       enabled: true
+       k8s:
+         service:
+           ports:
+             - port: 15021
+               targetPort: 15021
+               name: status-port
+             - port: 15012
+               targetPort: 15012
+               name: tls-xds
+             - port: 15017
+               targetPort: 15017
+               name: tls-webhook
+EOF
+{{< /text >}}
+
+Apply the configuration in `external_cluster` in the `istio-system` namespace:
+
+{{< text bash >}}
+$ istioctl apply -f external-cp.yaml --context="${CTX_EXTERNAL_CLUSTER}"
+{{< /text >}}
+
+Create the Istio network configuration to expose the **yet to be installed** external istiod on the ingress gateway in the `istio-system` namespace:
```

Comment:
DIdn't you already install istiod in the previous step?

Reply:
The first istiod is just to manage the ingress gw.  It was not an external istiod.

Reply:
That is really confusing. We can't let this go to alpha with this flow in place.